update to 5.0.16

jaime-ez opened the pull request:
Completes CORS options for uws
Complements monitoring description including log monitoring

yasserf commented on src/tutorials/integrations/other-heroku/readme.md:
````diff
@@ -0,0 +1,260 @@
+```
+
+Currently there is no heroku add-on or buildpack for deepstream,
+so we're going to use a heroku buildpack which based on Node.js.
````
is based*

yasserf commented on src/tutorials/integrations/other-heroku/readme.md:
````diff
@@ -0,0 +1,260 @@
+Now deepstream server can be installed via
+
+```shell
+npm install deepstream --save
````
deepstream.io?

yasserf commented on src/tutorials/integrations/other-heroku/readme.md:
````diff
@@ -0,0 +1,260 @@
+```javascript
+  //...
+  "engines": {
+    "node": "4.4.7"
````
does this support the term 'lts'?

If it does we could use that to make it more.. timeless

yasserf commented on src/tutorials/integrations/other-heroku/readme.md:
````diff
@@ -0,0 +1,260 @@
+git commit -a -m "init commit"
+```
+
+Fore more details how to use git, here is a [beginner guide](https://rogerdudler.github.io/git-guide)
````
for

jaime-ez committed:
[task] complet http monitoring decription and include log monitoring

jaime-ez committed:
[task] complete http monitoring decription and include log monitoring

:100644 100644 2960f0d 860b543 M	content/tutorials/60-plugins/60-monitoring/http/index.md

jaime-ez committed:
[task] uws server CORS options

:100644 100644 517d9e8 b452130 M	content/docs/10-server/10-configuration/index.md

jaime-ez committed:
[task] complete description

:100644 100644 860b543 f908be2 M	content/tutorials/60-plugins/60-monitoring/http/index.md

:100644 100644 860b543 f908be2 M	content/tutorials/60-plugins/60-monitoring/http/index.md

yasserf commented on content/tutorials/60-plugins/60-monitoring/http/index.md:
```diff
@@ -72,12 +72,33 @@ This provides us good insight into what the server is actually doing, but doesn'
 monitoring:
   type: http
+  # the endpoint url
+  url: /monitoring,
```
I think comas are not actually in the yaml spec, so this might be invalid (I think)

jaime-ez replied:
yes probably the comma slipped from my json config file
